instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/__init__.py

```diff
@@ -108,6 +109,18 @@ def _instrument(self, **kwargs):
                 kwargs.get("engine"),
                 kwargs.get("enable_commenter", False),
             )
+        if kwargs.get("engines") is not None and isinstance(
+            kwargs.get("engines"), list
```

Comment:
Better to check for a more [generic type](https://docs.python.org/3/library/collections.abc.html#collections.abc.Sequence).